src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/ScoringCard.test.jsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import {
-  render, screen, initializeMocks, fireEvent,
+  screen, initializeMocks, fireEvent,
 } from '@src/testUtils';
 import ScoringCard from './ScoringCard';
-import { selectors } from '../../../../../../data/redux';
+import { selectors, initializeStore } from '../../../../../../data/redux';
+import editorRender from '../../../../../../modifiedEditorTestRender';
```

Comment:
```suggestion
import { selectors } from '@src/editors/data/redux';
import { editorRender } from '@src/editors/editorTestRender';
```
Please remove `modifiedEditorTestRender` and you don't need to import `initializeStore`